### ISSUE TITLE
Question about parameters

### ISSUE THREAD

**Issue by Ben7124** [mhalushka/miRge3.0#50]
Hello,

In practical usage, which parameters are best? Is mirgenedb preferred to mirbase? Do the "ai" and "trf" flags improve detection? Do you recommend the use of these flags?
Thank you!

**Comment by mhalushka** [comment]
There is no 100% right answer as it depends on what you want to learn, but I'll do my best.  MirGeneDB is preferred over miRBase in almost every scenario.  If you are looking for a biomarker and you don't care if it is a bonafide miRNA or not, miRBase might be useful.  If you are interested in biology, then only MirGeneDB should be considered.  The -ai flag gets very little useful data, so that is probably not that helpful.  The -trf is good if you are also interested in tRF distributions in the data.  They don't improve detection.  The miREC method is good for curtailing a long tail of isomiRs, but comes at the cost of slowing down throughput through miRge. 
tl;dr MirGeneDB >> miRBase; don't use -ai; use trf if interested in tRFs.
